### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by RayeN**
Also:

 - make service initialization to look like other services
 - add change log
 - send json for all requests
 - add new dependencies to setup.py

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by bearnard** [comment]
:+1:

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```
Comment (stefanor) [inline setup.py]: Why would you remove the shebang?

Comment (RayeN) [inline setup.py]: And why do we need it?

Comment (stefanor) [inline setup.py]: So you can run setup.py directly: `./setup.py`

Comment (RayeN) [inline setup.py]: Yeah, but the file wasn't executable.  And I thought we never install it by running either `./setup.py` or `python setup.py` manually.  If there's case for that - I'll return it back

Comment (stefanor) [inline setup.py]: Right, yes, it has to be executable, too.

>  And I thought we never install it by running either ./setup.py or python setup.py manually

Well, we do upload it by running `setup.py sdist`, manually. And it's convenient to not have to type `python ...`

I'd say put the shebang back, and make it executable :)

Comment (RayeN) [inline setup.py]: Done, https://github.com/yola/sitewit/pull/12

**Pull request merged**